automatic/1password7/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,43 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName    = $env:ChocolateyPackageName
+  fileType       = 'exe'
+  url            = 'https://c.1password.com/dist/1P/win6/1PasswordSetup-7.9.835.exe'
+  softwareName   = '1Password*'
+  checksum       = '3d43ea28c46a1dac9cc645c5d9f4692cd1be009145ffe5260a910a2d9c89dd52'
+  checksumType   = 'sha256'
+  silentArgs     = "--silent"
+  validExitCodes = @(0)
+
+  BeforeInstall  = {
+    $cache_dir = Join-Path -Path $env:LocalAppData -ChildPath "1password\logs\setup"
+
+    # Installer blocks at the end and never returns. Successifull installation is visible in the log file, but only if it is the first installation
+    Start-Job -ScriptBlock { param($cache_dir)
```

Comment:
```suggestion
    # Installer blocks at the end and never returns. Successfull installation is visible in the log file, but only if it is the first installation
    $null = Start-Job -ScriptBlock { param($cache_dir)
```
Minor typo + not outputting the PSRemotingJob
![image](https://github.com/user-attachments/assets/7ae0759e-5c02-4f64-876f-de271d5acd44)